src/chart_types/xy_chart/renderer/canvas/xy_chart.tsx

```diff
@@ -78,6 +80,8 @@ export interface ReactiveChartStateProps {
   annotationSpecs: AnnotationSpec[];
   panelGeoms: PanelGeoms;
   seriesTypes: Set<SeriesType>;
+  customDescription: string | undefined;
```

Comment:
I think it's effectively equivalent but I think defining this as an optional prop communicates the intent a little better. 

```suggestion
  customDescription?: string;
```

Reply:
Smart! Thank you 3b13cc9